data/tasks.txt

```diff
@@ -2,3 +2,4 @@ T | 0 | sleep
 D | 0 | 2103T ip| 28 of August 2019, 11:59 PM
 E | 1 | project meeting| Friday 2101 classroom
```

Comment:
As this is a local storage file, it might not be a good idea to commit this file, as it could potentially contain sensitive information. You can add the relative directory of this file to .gitignore file, so it will be ignored by git and not be committed.